test-cases/features/tombstone_gc/longevity-tombstone-gc-modes.yaml

```diff
@@ -0,0 +1,19 @@
+test_duration: 200
+
+stress_cmd: ["scylla-bench -workload=timeseries -mode=write -replication-factor=3 -partition-count=400 -clustering-row-count=10000 -clustering-row-size=200 -concurrency=10 -rows-per-request=1 -start-timestamp=SET_WRITE_TIMESTAMP -connection-count 10 -max-rate 30000 -timeout=120s -retry-number=30 -retry-interval=80ms,1s -duration=80m"]
+
+n_db_nodes: 3
+n_loaders: 1
+n_monitor_nodes: 1
+
+instance_type_db: 'i3.large'
+
+
+user_prefix: 'longevity-tombstone-gc-modes'
+
+# Table settings:
+# gc_grace_seconds: 4 minutes
+# TTL: 5 minutes
```

Comment:
are you sure TTL is 5 min? where is it set?

Reply:
it's in the yaml file:
```
post_prepare_cql_cmds: "ALTER TABLE scylla_bench.test with gc_grace_seconds = 240 and default_time_to_live = 240 and compaction = {'class':'TimeWindowCompactionStrategy', 'compaction_window_size': 5, 'compaction_window_unit': 'MINUTES'} and tombstone_gc = {'mode': 'disabled', 'propagation_delay_in_seconds':'240'};"
```

Reply:
> it's in the yaml file:
> 
> ```
> post_prepare_cql_cmds: "ALTER TABLE scylla_bench.test with gc_grace_seconds = 240 and default_time_to_live = 240 and compaction = {'class':'TimeWindowCompactionStrategy', 'compaction_window_size': 5, 'compaction_window_unit': 'MINUTES'} and tombstone_gc = {'mode': 'disabled', 'propagation_delay_in_seconds':'240'};"
> ```

the TTL in the command is not set to 5 minutes... please fix either the comment, or the command